### ISSUE TITLE
Feature request: Grayscale color

### ISSUE THREAD

**Issue by r4dixx** [thompsonate/Shifty#92]
In Catalina, system color tint presets are here, including a greyscale one. Maybe this might be a fit? This would be awesome to have such option. Triggered after a certain time or even (and maybe I'm overthinking it) linked to the new Screen Time feature. What do you think?

Edit: Just like in Android with the Digital Wellbeing system app: https://play.google.com/store/apps/details?id=com.google.android.apps.wellbeing. By the way, love this app. Thanks for doing it! I'm available for testing if you need some :)

_Originally posted by @r4dixx in https://github.com/thompsonate/Shifty/issues/85#issuecomment-539234600_

**Comment by saagarjha** [comment]
This, maybe?

<img width="316" alt="Screen Shot 2019-10-14 at 01 57 34" src="https://user-images.githubusercontent.com/13786931/66739623-fe59a800-ee25-11e9-8d88-611ba5a9fbb0.png">

**Comment by choco** [comment]
Wasn't that already introduced in Mojave? Also, those don't influence Night Shift as far as I know?

I haven't really dug again in the framework itself, but I don't think they made many changes regarding different color profiles, so I would probably close this issue and consider it out of scope just like #85

**Comment by saagarjha** [comment]
Yes and yes. I think the request might have just been to change the system accent color at night.

**Comment by r4dixx** [comment]
No, I'm talking about this

![Improvements for the blind and hearing-impaired, from Engadget](https://o.aolcdn.com/images/dims?crop=1320%2C942%2C0%2C0&quality=85&format=jpg&resize=1600%2C1142&image_uri=https%3A%2F%2Fs.yimg.com%2Fos%2Fcreatr-uploaded-images%2F2019-06%2F809e23a0-9623-11e9-8479-d4561c747cc8&client=a1acac3e1b3290917d92&signature=4a7b8d7baae02733c1e551e4524d51ef8ac45eaa)

However, after a bit of research I noticed this might not be new (it just moved). But I still definitely think that a grayscale option like this would fit.

See

Sorry if it wasn't clear.

**Comment by r4dixx** [comment]
Hi! Any news about this?

**Comment by choco** [comment]
Hi, personally I don't fell this like fits the project, but I'll keep the issue open anyway if someone wants to work on this.
